m-21-32/Fileoperations
# Integrate MongoDB for data storage

# Mini-Project 7: MongoDB Basics

## Changes Made

- Integrated MongoDB with the existing React application.
- Created collections and documents in MongoDB.
- Updated React components to fetch and post data to MongoDB.
- Added error handling for data interactions with MongoDB.
- Addressed edge cases for empty responses and unexpected data.

## How to Test

1. Ensure MongoDB is installed and running locally.
2. Clone the repository and switch to the `mini-project-7` branch.
3. Install dependencies using `npm install`.
4. Start the React app using `npm start`.
5. Navigate to `/my-component` and interact with the MongoDB-integrated features.
